index.html

```diff
@@ -0,0 +1,168 @@
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    <div class="hamburger">
+      <i class="fas fa-bars fa-2x" id="ham_btn"></i>
+    </div>
+    <div class="nav_logo" id="main_nav">
+     <div>
+
+        <a href="#"><img src="./Imgs/logo.png" alt="Logo"></a>
+      </div>
+
+      <nav>
+        <ul>
+          <li class="header-link"><a href="#" class="current">Home</a></li>
+          <li class="header-link"><a href="about.html">About</a></li>
+          <li class="header-link"><a href="#project_schedule">Program</a></li>
+          <li class="header-link"><a href="#">Previous Events</a></li>
+          <li class="header-link"><a href="#art_guests">Guests</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+
+  <main>
+    
+
+    <section id="head_section">
+      <div class="art_head">
+        <div class="intro">
+          <h1 class="hello">"Hello! BEIRUT ART FAIR "</h1>
+          <h2 class="Caption">International modern and contemporary art fair</h2>
+        </div>
+
+        <div class="intro_parag">BEIRUT ART FAIR is the space for artistic revelations in the region.
+             A major event in the art fair calendar, the fair attracts a growing number of collectors, institutions,
+             art professionals and visitors from around the world, totalling 36,000 entries in 2019.
+        </div>
+
+        <div class="daate">
+          <h2><i class="far fa-calendar-alt"></i> 221.9.20(MON) Till 28(TUE)</h2>
+          <h3><i class="fas fa-map-marker-alt"></i> Seaside Arena halls 1 & 2, Beirut New Waterfront, Beirut, Lebanon</h3>
+        </div>
+      </div>
+    </section>
+
+    
+    <section id="project_schedule">
+        <div class="main-program">
+          <h3>Main Program</h3>
+          <hr>
+        </div>
+  
+        <div class="art_programs">
+          <div class="program">
+            <i class="fa fa-paint-brush fa-2x"></i>
+            <h4>Meet the art</h4>
+            <p>LEBANON MODERN!  Unexpected Trove - The Unseen Works of Hussein Madi (Rome 1964-1970)n</p>
+          </div>
+          <div class="program program_2">
+            <i class="fas fa-child fa-2x"></i>
+            <h4>KIDS WORKSHOPS</h4>
+            <p>Enjoy art with your family at the fair! Dedicated tours will be available for children aged 6–12 Given by education professionals</p>
+          </div>
+          <div class="program program_2">
+            <i class="fas fa-microphone-alt"></i>
+            <h4>TALKS AND ROUNTABLES</h4>
+            <p>Our program of roundtables and talks aims at opening up the discussion of the past
+                 and future circumstances of various MENA art scenes in constant evolution.</p>
+          </div>
+
+  
+        </div>
+  
+        <div class="program_btn">
+          <a href="#">See the whole program</a>
+        </div>
+
```

Comment:
- [ ] OPTIONAL
Your code is not properly indented. Kindly ensure that your code has consistent **spacing** for indentation. The recommended spacing is **2 spaces**. You can install and configure a code formatter plugins such as [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) in your code editor for formatting your code to ensure consistent and well-indented code.